java/kotlin-extractor2/src/main/kotlin/entities/Class.kt

```diff
@@ -183,24 +183,24 @@ fun KotlinFileExtractor.extractClassSource(
     }
 }
 
-/*
-    OLD: KE1
-        // `args` can be null to describe a raw generic type.
-        // For non-generic types it will be zero-length list.
-    */
+data class ClassLabelResults(val classLabel: String, val shortName: String)
+
+// `args` can be null to describe a raw generic type.
+// For non-generic types it will be zero-length list.
+// TODO: raw types not yet implemented for KE2
 context(KaSession)
 private fun KotlinUsesExtractor.getClassLabel(
     c: KaClassSymbol,
-    argsIncludingOuterClasses: List<Nothing>?
+    argsIncludingOuterClasses: List<List<KaTypeProjection>>?
```

Comment:
I think it would be worth including a comment with a small example explaining what this list-of-lists means.